src/EnergyPlus/SingleDuct.cc

```diff
@@ -4800,7 +4800,7 @@ namespace SingleDuct {
         MassFlow = sd_airterminalInlet(SysNum).AirMassFlowRate; // system air mass flow rate
 
         if (GetCurrentScheduleValue(this->SchedPtr) > 0.0 && MassFlow > SmallMassFlow) {
-            Real64 CpAir = PsyCpAirFnW(0.5 * (Node(this->OutletNodeNum).HumRat + Node(ZoneNodeNum).HumRat));
+            Real64 CpAir = PsyCpAirFnW(Node(this->OutletNodeNum).HumRat);
```

Comment:
Changed to make it consistent with the SensOutputProvided variable computed in AirDistributionUnit.

Reply:
Note that the zone air node humidity ratio used in the previous equation is at previous time step.

Reply:
But the previous time step zone humidity ratio and zone temperature are the conditions predictor uses to calculate the zone sensible and latent loads. I thought Cp was the average or integrated value between those 2 conditions (outlet and zone or inlet). Whatever method is used to calculate delivered capacity must also be used by the parent equipment (e.g., Furnace, UnitarySystem) because remaining load is calculated with respect to the terminal unit delivered capacity.

Reply:
This fix was intended to make delivered sensible output calculation consistent among air terminal units. Otherwise, there are different ways how the sensible output is computed. Perhaps Cp of moist air calculated at average humidity ratio is the most accurate of all approaches. Furthermore, we need to make sure that the fix addresses that sensible and latent outputs sum is equal to the total delivered (heating or cooling) rate. @rraustad I will revisit this fix.